dart/lib/src/sentry_options.dart

```diff
@@ -228,6 +232,16 @@ class SentryOptions {
   /// Whether to send personal identifiable information along with events
   bool sendDefaultPii = false;
 
+  /// Configures whether to record exceptions for failed requests.
+  /// Examples for captures exceptions are:
+  /// - In an browser environment this can be requests which fail because of CORS.
+  /// - In an mobile or desktop application this can be requests which failed
+  ///   because the connection was interrupted.
+  bool captureFailedRequests = false;
+
+  /// Whether to records requests as breadcrumbs. This is on by default.
+  bool recordHttpBreadcrumbs = true;
```

Comment:
Maybe add that this only has an effect when the SentryHttpClient or dio integration is in use. Same for all the other http related settings.